scripts/previous-releases.jq

```diff
@@ -0,0 +1,23 @@
+# Extract only the `tag_name` field from each release object
+[ .[].tag_name ]
+
+# Transform each tag name into a structured object using regex capture
+| reduce map(
+    capture("^(?<full>release(-(?<component>proxy|compute))?-(?<release>\\d+))$")
+    | {
+        component: (.component // "storage"),  # Default to "storage" if no component is specified
+        release: (.release | tonumber),        # Convert the release number to an integer
+        full: .full                             # Store the full tag name for final output
```

Comment:
Fix the additional space in this line and move the jq program into `.github/scripts`